src/common/types/clusters.ts

```diff
@@ -24,9 +24,18 @@ export type HostSubnet = {
   humanized: string;
 };
 export type HostSubnets = HostSubnet[];
-export type NetworkConfigurationValues = V2ClusterUpdateParams & {
```

Comment:
I opted for more defensive approach. Instead of including all `V2ClusterUpdateParams`, i just pick the ones we really use.